Vehicle.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CodeReviewDemo
+{
+    internal class Vehicle : IVehicle
+    {
```

Comment:
Consider using PascalCase for the `model` and `year` properties to maintain consistency with C# naming conventions. The correct naming should be `Model` and `Year`.